src/extractor/guidance/turn_lane_handler.cpp

```diff
@@ -352,14 +351,8 @@ bool TurnLaneHandler::isSimpleIntersection(const LaneDataVector &lane_data,
             if (lane_data.back().tag == TurnLaneType::uturn)
                 return findBestMatchForReverse(lane_data[lane_data.size() - 2].tag, intersection);
```

Comment:
Can you add more context here why you need to find a match for `lane_data[-2]` if `lane_data[-1]` is a uturn?
I think it would help to use named variables here for documentation.